docs/whitepaper/whitepaper.md

```diff
@@ -0,0 +1,591 @@
+
+# 2. Example Use Cases
+
+Specific use cases that we intend to support.
+The core idea is to support as many use-cases as possible by enabling interoperability
+between a large variety of ledgers specific to certain main stream or exotic use cases.
+
+## 2.1 Fabric to Quorum Asset Transfer
+
+Export an asset from one network to the other.
+Details TBD
+
+## 2.2 Escrowed Sale of Data for Coins
+
+Organization A is looking to buy some data. Organization B has some data it's looking to monetize.
+
+Data in this context is any series of bits stored on a computer:
+* Machine learning model
+* ad-tech database
+* digital/digitized art
+* proprietary source code or binaries of software
+* etc.
+
+Organization A and B trade the data and the funds through a BIF transaction in an
+atomic swap with escrow securing both parties from fraud or unintended failures.
+
+Through the transaction protocol's handshake mechanism, A and B can agree (in
+advance) upon
+* The delivery addresses (which ledger, which wallet)
+* the provider of escrow that they both trust
+* the price and currency
+
+Establishing trust (e.g. Is that art original or is that machine learnig model has the advertized accuracy) can be facilitated through the participating DLTs if they support it.
+
+## 2.3 Money Exchanges
+
+Enabling the trading of fiat and virtual currencies in any permutation of possible pairs.
+
+## 2.4 Stable Coin Pegged to Other Currency
+
+Someone launches a highly scalable ledger with their own coin called ExampleCoin that can consistently sustain throughput levels of a million transactions per second reliably, but they struggle with adoption because nobody wants to buy into their coin fearing that it will lose its value. They choose to put in place a two-way peg with Bitcoin which guarantees to holders of their coin that it can always be redeemed for a fixed number of Bitcoins/USDs.
+
+### 2.4.1 With Permissionless Ledgers (BTC)
+
+A BTC holder can exchange their BTC for ExampleCoins by sending their BTC to `ExampleCoin Reserve Wallet` and the equivalent amount of coins get minted for them
+onto their ExampleCoin wallet on the other network.
+
+An ExampleCoin holder can redeem their funds to BTC by receiving a Proof of Burn on the ExampleCoin ledger and getting sent the matching amount of BTC from the `ExampleCoin Reserve Wallet` to their BTC wallet.
+
+![ExampleCoin Pegged to Bitcoin](https://www.plantuml.com/plantuml/png/0/XP9FIyH03CNlyoc2dhmiui6JY5ln7tWG4KJmaft6TkWqgJFfrbNyxgPBLzP5yLQQlFpoNkOiAoRjsmWNRzXsaSubCDnHLL49Ab04zVR7EGqQ2QvN7QL8PKK9YYY-yJLQ_mqhLGar2CDbmfO6ISqp_pCoDu4xj7R8zDeJUvgd9CD37Np3b3CSRRKawRdqajZ8HuTRXHRVMcl6Yd9u9pW-_6NkdNaCFdJ82ZR6B0Gcvrx6LM7lGHH_-h_X9R5AMkq1Pb3o0VPlGrNhLS8LV3W0bjAKyuCViaUCaJIlHUI7RmqHhqMVxdC7EcMn2rpynOiHHEin_4cuZKHPR9XF5ACC4tIZBWvsZmptb2ajAKzpfisxzCVkewcJsMnskcbQrnsB4jZfBTN6pG6vX08iUZDed2N6dc117ljChe2GOO7URbI1MPdAyW9po09Hk79Z15OPrZj1BT4kDieGDFLPxHbPM45NCqU66kdEdTcdFUCl "ExampleCoin Pegged to Bitcoin")
+
+#### 2.4.1.1 Proof of Burn
```

Comment:
Out of curiosity, does anyone have imminent plans for proof of burn cryptocurrencies?  To my knowledge there aren't a lot of popular cryptocurrencies that use this.

Reply:
I would also point out that this is inconsistent with the objective that the number of assets in a chain not be modified (3.1.10)

Reply:
Discussed in Feb 3 contributors to remove as unnecessary

Reply:
I don't know about any implementations either.
Thank you all for the feedback, as discussed I removed the section regarding proof of burn.